tests/helper/tests/rkhunter.py

```diff
@@ -0,0 +1,52 @@
+import os
+from helper import utils
+import logging
+
+logger = logging.getLogger(__name__)
+
+def rkhunter(client, testconfig):
+    """Run rkhunter to test for rootkits"""
+    utils.AptUpdate(client)
+    (exit_code, output, error) = client.execute_command(
```

Comment:
We can simply use `install_package_deb()` from utils. (see more: https://github.com/gardenlinux/gardenlinux/blob/e94888ba4049598d252111e00f22eef0788d5caf/tests/helper/tests/orphaned.py#L10)